helix-query.yaml

```diff
@@ -22,3 +22,156 @@ indices:
         select: head > meta[property="og:url"]
         value: |
           match(attribute(el, 'content'), 'https:\/\/[^/]+(\/.*)')
+
+  seo-au:
+    <<: *seo
+    include:
+      - 'au/acrobat/**'
+      - 'au/creativecloud/**'
+      - 'au/sign/**'
+    exclude:
+      - 'de/**'
```

Comment:
I think so; by default it should include all that's defined in https://github.com/adobe/fedpub/blob/main/fstab.yaml#L2

Reply:
> Does it need the excludes?

Actually it doesn't, sorry, it that isn't clear from the documentation. The rule is: only add things to that index that are defined in the *include* section, **unless** they're listed in the *exclude* section. In other words: you don't need to exclude `de/**` because it doesn't match any of the items in the include section.